misc/collect.py

```diff
@@ -78,6 +78,6 @@
       exit(1)
 
 if args.formatted:
-  json.dump(output, sys.stdout, indent="  ")
```

Comment:
Good idea. I removed it because the the release list should not be sorted. But that is not needed anymore.